src/payform.coffee

```diff
@@ -29,11 +29,19 @@
 
   _eventNormalize = (listener) ->
     return (e = window.event) ->
-      e.target = e.target or e.srcElement
-      e.which = e.which or e.keyCode
-      unless e.preventDefault?
-        e.preventDefault = -> this.returnValue = false
-      listener(e)
+      newEvt =
```

Comment:
Are these handpicked attributes? Should we clone all attributes with a polyfill of `Object.keys` + the `preventDefault` assign or indeed go with only these ones?

Reply:
These were handpicked by surfacing the attributes used across the library.

While the recreated object isn't exposed, handpicking means that we <b>must</b> make sure that we do not omit any other used event property.

Using the standard `Object` wrapper (i.e. `clone()`, `defineProperty()`) to clone events and handle properties provides us with better assurance. However, during my last discussion, opting for a simple synthetic event without any polyfill was suggested.